section1/reflection.md

```diff
@@ -1,19 +1,101 @@
 ## Section 1 Reflection
```

Comment:
Really great work in Section 1 and I appreciate how thorough you were with this reflection. 

Tadpoles - that is so cute!! My app is just called KinderCare.